lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -252,8 +253,8 @@ def test_text_rotated_urls():
 
     test_url = 'https://test_text_urls.matplotlib.org/'
 
-    fig = plt.figure(figsize=(2, 1))
-    fig.text(0.1, 0.1, 'test plain 123', rotation=45, url=f'{test_url}')
+    fig = plt.figure(figsize=(1, 1))
+    fig.text(0.1, 0.1, 'N', rotation=45, url=f'{test_url}')
```

Comment:
This change was not really on purpose, but I guess there is no need to change it.

(The rotation is not really obvious and my idea was that by having letter that started with a "full height" bar, I could more easily detect the effect, but that didn't really happen... Forgot to change it back though...)